.pre-commit-config.yaml

```diff
@@ -164,8 +164,8 @@ repos:
     - --html-report=.tox/.tmp/.mypy/python-3.8
     pass_filenames: false
   - id: mypy
-    alias: mypy-py37
-    name: MyPy, for Python 3.7
+    alias: mypy-py38
```

Comment:
This is a duplicate, just drop it.

Reply:
Also, check if it's already possible to run it against 3.13 and add it to the top. In general, it's good to run against every other version where possible. This would need an update in the reusable linters workflow, if added.

Reply:
Ah, didn't realise it was running under multiple versions. Seems a bit overkill for Mypy..

Reply:
This helps cover interpreter-conditional branches. It's cheap enough.